primitive/src/main/java/io/atomix/primitive/partition/impl/PrimaryElectorService.java

```diff
@@ -103,6 +103,7 @@ protected void configure(ServiceExecutor executor) {
   }
 
   private void notifyTermChange(PartitionId partitionId, PrimaryTerm term) {
+    getLogger().debug("Changed " + term + ", listeners=" + listeners.size());
```

Comment:
We should always use string replacement with `{}`

Reply:
yes, sorry, I will fix this and update the PR